anvio/tests/run_all_tests.sh

```diff
@@ -656,6 +655,24 @@ anvi-dereplicate-genomes --ani-dir $output_dir/GENOME_SIMILARITY_OUTPUT \
                          --program pyANI
 SHOW_FILE $output_dir/DEREPLICATION_FROM_PREVIOUS_RESULTS/CLUSTER_REPORT.txt
 
+INFO "Testing anvi-analyze-synteny ignoring genes with no annotation"
+
+# make a external-genomesfile
+echo -e "name\tcontigs_db_path\ng01\t$output_dir/01.db\ng02\t$output_dir/02.db\ng03\t$output_dir/03.db" > $output_dir/external-genomes-file.txt
+
+# run anvi-analyze-synteny
+anvi-analyze-synteny -e $output_dir/external-genomes-file.txt \
+          --annotation-source COG_FUNCTION \
+          --window-range 2:3 \
+          -o $output_dir/synteny_output_no_unknowns.tsv
```

Comment:
This,

```
anvi-analyze-synteny -e $output_dir/external-genomes-file.txt \
          --annotation-source COG_FUNCTION \
          --window-range 2:3 \
          -o $output_dir/synteny_output_no_unknowns.tsv
```

Should be like this for compliance with the rest of the file (again, small things, but they count):

```
anvi-analyze-synteny -e $output_dir/external-genomes-file.txt \
                     --annotation-source COG_FUNCTION \
                     --window-range 2:3 \
                     -o $output_dir/synteny_output_no_unknowns.tsv
```